src/__tests__/fieldValidation.unit.test.ts

```diff
@@ -58,19 +50,4 @@ describe('field value validation against BaseFieldDataType', () => {
 			false,
 		);
 	});
-	test('validate a valid URL as URL', () => {
```

Comment:
I suspect that removing these tests entirely will result in less code coverage -- let's make sure the basic URL  positive and negative case is *covered* by our code, even if we don't need to exhaustively test all the possible edge cases.

Likewise for the email tests.

Basically, +1 to removing a bunch of these `expects` but let's keep just one in each for the sake of ensuring we didn't completely space out on our testing.